css/styles.css

```
@@ -71,19 +62,48 @@ body {
   background-color: var(--basic-white);
 }
 
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
 .list-reset {
   margin: 0;
   padding: 0;
   list-style: none;
 }
 
+/*GRID*/
+.page-body {
+  min-height: 100%;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  align-content: start;
+}
+
 /* MAIN NAVIGATION */
-.main-navigation {
+.main-header {
   font-weight: 500;
   line-height: 30px;
   background-color: var(--white-grey);
 }
 
+.main-navigation {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  width: 1200px;
+  margin: 0 auto;
+  padding: 0 20px;
```

чтобы не писать каждый раз эти стили для центровки содержимого, создай класс `container` и используй его:
.container {
```suggestion
  width: 1200px;
  margin: 0 auto;
  padding: 0 20px;
```
  width: 1200px;
  margin: 0 auto;
  padding: 0 20px;
}